doc/performance_blog_post/performance_blog_post.md

```diff
@@ -0,0 +1,120 @@
+
+## Displaying Performance Tests Results
+
+In order to evaluate the test results, and discuss them, we need to visualize them. For that we decided for a [Redash](https://redash.io/) dashboard. Redash is an open source tool that allows you to create online dashboards for SQL queries. So we push our performance test results into an Exasol database and use Redash to evaluate them.
+
+## Setting Goals
+
+The first test run generated the following results:
+
+| Test Case                          | Time in Seconds |
+|------------------------------------|-----------|
+| testLoadManyParquetRows            | 23        |
+| testLoadLargeParquetRows           | 13.02     |
+| testLoadManyParquetColumns         | 12.9      |
+| testLoadManyParquetRowsFromOneFile | 74.64     |
+| testManySmallJsonFiles             | 685.87    |
+| testLoadFewBigSalesParquetFiles    | 662.28    |
+| testLoadSalesParquetFiles          | 219.21    |
+
+The first 4 cases (`testLoadManyParquetRows`, `testLoadLargeParquetRows`, `testLoadManyParquetColumns` `testLoadManyParquetRowsFromOneFile`) all load the same amount of data (1 GB). They just differ on how the data is distributed over rows, files and columns.
+
+This dataset consists of pseudo-random generated strings. The different test cases are built as show in the following table:
+
+| Test Case                          | String size in byte | Rows      | Files | Columns |
+|------------------------------------|---------------------|-----------|-------|---------|
+| testLoadManyParquetRowsFromOneFile | 1,000               | 1,000,000 | 1     | 1       |
+| testLoadManyParquetRows            | 100                 | 1000000   | 10    | 1       |
+| testLoadLargeParquetRows           | 1,000,000           | 100       | 10    | 1       |
+| testLoadManyParquetColumns         | 1000                | 1000      | 10    | 100     |
+
+You can observe that the product of all columns is always 1,000,000,000 so 1 GB.
+
+Similar the two trailing test split the same test data in 200 (`testLoadSalesParquetFiles`) or 8 `testLoadFewBigSalesParquetFiles`. There the data is exemplary sales data consisting of 7 decimal columns with 3,621,459,710 rows.
+
+![Performance test results of Parquet file loading 1](slowParquetTestResults1.png)
+![Performance test results of Parquet file loading 2](slowParquetTestResults2.png)
+
+From the charts, it becomes clear that the Virtual Schema at that point has an issue with loading data from few big files. When loading from more files, it loads the same amount of data a lot quicker.
+
+Starting with this analysis, we discussed with our product management and decided to invest into performance improvements for loading big Parquet files. More precisely we set the goal to make improvements so that the `testLoadFewBigSalesParquetFiles` also runs in less than 250s.
```

Comment:
```suggestion
Starting with this analysis, we discussed with our product management and decided to invest in performance improvements for loading big Parquet files. More precisely we set the goal to make improvements so that the `testLoadFewBigSalesParquetFiles` also runs in less than 250s.
```